ansible/roles/opensearch/templates/opensearch.yml.j2

```diff
@@ -0,0 +1,34 @@
+### This section copies a running container's default config ###
```

Comment:
Should this be a template if it doesn't have any template strings in?

Reply:
Hmm not really, but I'm not sure if the `copy` module searches inside a role?